src/tools/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+
+  <Import Project="..\..\Directory.Build.props" />
+
+  <PropertyGroup>
+    <SlnGenSolutionFolder>tools\</SlnGenSolutionFolder>
```

Comment:
> Is this related to the "tools" paths that are showing up in the generated solutions?

Yes. With that change, the folder path for MonoLinker in the sln is `tools\src\Mono.Linker.csproj`. The subpaths under tools (i.e. `src`) are necessary because Mono.Linker.csproj exists twice (ref & src) and VS needs them to be in different subfolders.